hr_attendance_autoclose/README.rst

```diff
@@ -0,0 +1,90 @@
+========================
+HR Attendance Auto Close
+========================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fhr-lightgray.png?logo=github
+    :target: https://github.com/OCA/hr/tree/10.0/hr_attendance_autoclose
+    :alt: OCA/hr
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/hr-10-0/hr-10-0-hr_attendance_autoclose
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/116/10.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module close stale attendances automatically. Without this module, when
+an employee forgets to check out at the end on the day and the next day the
+employee does not realize of that, then the error is propagated and all the
+attendances are wrong. With this module only the attendance with the issue is
+wrong, and the manager know the system closes the attendance, not the employee
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+#. Go to *Attendances > Manage Attedances > Attendances*.
+#. Go to *Attendances > Manage Attedances > Attendances*.
+#. Attendance are autoclosed after the value specified.
+
+Known issues / Roadmap
+======================
+
+* Known Issues
+
+This module overwrites the standard behavior of worked_hours field in
```

Comment:
I think there's no overlap as the name of the field is different. Any way, I would say to use a more explicit name for the field.

Reply:
Actually I am using the same field now. I think it is no need for duplicating the field. Is it?

Reply:
Making the standard field not stored was wrong so I go back to the previous approach and use a better field name.

Reply:
Removed from readme

Reply:
Is ok `open_worked_hours` as name for the new field?